controllers/reviews.controllers.js

```diff
@@ -10,4 +11,13 @@ exports.getReviewById = (req, res, next) => {
     selectReviewById(req.params.review_id).then((review) => {
         res.status(200).send({ review })
     }).catch(next)
+}
+
+exports.patchReview = (req, res, next) => {
+    Promise.all([
```

Comment:
do you need to check if the review exists in this case, I think you should be a blessing to handle this in the updateReview model